### PR TITLE
Update hello_world.html

### PR DESCRIPTION
**Pull request by kvhuang23**
Please review

### CODE REVIEW

File: hello_world.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
     <h1>Hello World</h1>
+    <h1>Bonjour Mode</h1>
```
Comment (kvhuang1) [inline hello_world.html]: Typo here.

**Review by kvhuang1 — CHANGES_REQUESTED** [review]
Please fix the typo

File: hello_world.html (hunk 1 above)
Comment (kvhuang23) [reply hello_world.html]: Fixed the typo. Please verify.

**Review by kvhuang1 — APPROVED** [review]
Looks good.

**Pull request merged**